tests/BigQuery/BigQueryClientTest.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests\BigQuery;
+
+use Google\Cloud\BigQuery\BigQueryClient;
+use Prophecy\Argument;
+
+class BigQueryClientTest extends \PHPUnit_Framework_TestCase
+{
+    public $connection;
+    public $jobId;
+    public $projectId;
+    public $client;
+
+    public function setUp()
+    {
+        $this->connection = $this->prophesize('Google\Cloud\BigQuery\Connection\ConnectionInterface');
```

Comment:
Class name references in the tests can be simplified. On PHP >= 5.5 (which the composer.json requires), the reference style (`ConnectionInterface::class`) returns a string for the FQCN (`'Google\Cloud\BigQuery\Connection\ConnectionInterface'`)